components/homepage/About/About.jsx

```diff
@@ -62,4 +62,4 @@ const About = () => {
   );
 };
 
-export default About;
+export default About;
```

Comment:
revert this file change

Reply:
i am sorry i didn't get it! what to revert in this? this is the same file currently present in the original repository..

I think i couldn't follow what you mean? could you please explain

Reply:
I think you have removed the next empty line. If you are unable to see the changes run following in sequence:
1. `git reset HEAD~`
2. `git add components/homepage/Footer/Footer.jsx`
3. `git add components/Navbar/Navbar.jsx`
4. `git commit -m '<MESSAGE>'`
5. `git push origin <BRANCH_NAME>`

Happy to help more if required